src/CMakeLists.txt

```diff
@@ -35,7 +35,7 @@ add_library(NUClear::nuclear ALIAS nuclear)
 # Set compile options for NUClear
 target_link_libraries(nuclear ${CMAKE_THREAD_LIBS_INIT})
 set_target_properties(nuclear PROPERTIES POSITION_INDEPENDENT_CODE ON)
-target_compile_features(nuclear PUBLIC c_std_99 cxx_std_14)
+target_compile_features(nuclear PUBLIC cxx_std_14)
```

Comment:
SonarQube complains about this? OR is this part of the xxhash changes?

Reply:
It was a part of xxhash, the c files are gone